tests/unit/FmCorrection.spec.ts

```diff
@@ -126,12 +115,31 @@ describe('Firm Correction component', () => {
         data: { business: { legalType: 'SP' } }
       }))
 
+    // GET auth info
+    get.withArgs('https://auth.api.url/entities/FM1234567')
+      .returns(Promise.resolve({
+        data: {
+          contacts: [
+            {
+              email: 'mock@example.com',
+              phone: '123-456-7890'
+            }
+          ]
+        }
+      }))
+
     // GET addresses
     get.withArgs('businesses/FM1234567/addresses')
       .returns(Promise.resolve({
         data: {}
       }))
 
+    // GET parties
+    get.withArgs('businesses/FM1234567/parties')
+      .returns(Promise.resolve({
+        data: { parties: [] }
+      }))
```

Comment:
So this component _does_ need this network mock, for fetching the business snapshot.

Reply:
I was a little to heavy handed in removing stuff I guess

Reply:
Hehehe, I forgot about these too.

What I noticed was warnings while running the tests. I think the tests for all of our UIs run cleanly so if there's anything amiss then I try to fix it.